fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/ExpressionTranslator.java

```diff
@@ -402,4 +414,61 @@ private static org.apache.doris.analysis.AssertNumRowsElement.Assertion translat
                 throw new AnalysisException("UnSupported type: " + assertion);
         }
     }
+
+    private void translateEncryptionFunctions(ScalarFunction function, PlanTranslatorContext context,
+            List<Expr> arguments, List<Type> argTypes) {
+        String blockEncryptionMode = context.getRuntimeTranslator().get().getRuntimeFilterContext()
+                .getSessionVariable().getBlockEncryptionMode();
```

Comment:
if u want session variables, u has two way to do that:
1. ConnectContext.get(). getSessionVariable()
2. put cascadesContext into PlanTranslatorContext